flow/layers/color_filter_layer_unittests.cc

```diff
@@ -187,32 +205,44 @@ TEST_F(ColorFilterLayerTest, Nested) {
   EXPECT_EQ(mock_layer1->parent_matrix(), initial_transform);
   EXPECT_EQ(mock_layer2->parent_matrix(), initial_transform);
 
-  SkPaint filter_paint1, filter_paint2;
-  filter_paint1.setColorFilter(layer_filter1);
-  filter_paint2.setColorFilter(layer_filter2);
-  layer1->Paint(paint_context());
-  EXPECT_EQ(
-      mock_canvas().draw_calls(),
-      std::vector({MockCanvas::DrawCall{
-                       0, MockCanvas::SaveLayerData{children_bounds,
-                                                    filter_paint1, nullptr, 1}},
-                   MockCanvas::DrawCall{
-                       1, MockCanvas::DrawPathData{child_path1, child_paint1}},
-                   MockCanvas::DrawCall{
-                       1, MockCanvas::SaveLayerData{child_path2.getBounds(),
-                                                    filter_paint2, nullptr, 2}},
-                   MockCanvas::DrawCall{
-                       2, MockCanvas::DrawPathData{child_path2, child_paint2}},
-                   MockCanvas::DrawCall{2, MockCanvas::RestoreData{1}},
-                   MockCanvas::DrawCall{1, MockCanvas::RestoreData{0}}}));
+  DisplayListBuilder expected_builder;
+  /* ColorFilterLayer::Paint() */ {
+    DlPaint dl_paint;
+    dl_paint.setColorFilter(dl_color_filter.get());
+    expected_builder.saveLayer(&children_bounds, &dl_paint);
+    {
+      /* MockLayer::Paint() */ {
+        expected_builder.drawPath(child_path1,
+                                  DlPaint().setColor(DlColor::kYellow()));
+      }
+      /* ColorFilter::Paint() */ {
+        DlPaint dl_paint;
```

Comment:
Please use a different name for the DlPaint variable here. We shouldn't have name collisions for local variables even if they are in a different scope and the compiler allows it.